webpack.config.js

```diff
@@ -291,7 +291,7 @@ module.exports = {
 		new DependencyExtractionWebpackPlugin( { injectPolyfill: true } ),
 	].filter( Boolean ),
 	watchOptions: {
-		ignored: '!packages/*/build-module/**/*',
+		ignored: [ '**/node_modules', '**/packages/*/src' ],
```

Comment:
I like it! Should it be `'**/packages/*/src/**'` though? Not tested so I'm not sure either.

Reply:
I wrote if after webpack docs and what @gwwar shared, we can make it more explicit just to be sure 😄

Reply:
Ahh, seems like specifying directory also works 👍

Reply:
nice, this is much more readable!